src/main/java/com/unascribed/fabrication/mixin/c_tweaks/feather_falling_no_trample/MixinFarmBlock.java

```diff
@@ -0,0 +1,38 @@
+package com.unascribed.fabrication.mixin.c_tweaks.feather_falling_no_trample;
+
+import com.unascribed.fabrication.support.EligibleIf;
+import net.minecraft.block.Block;
+import net.minecraft.block.FarmlandBlock;
+import net.minecraft.enchantment.EnchantmentHelper;
+import net.minecraft.enchantment.Enchantments;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.GameRules;
+import net.minecraft.world.World;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Overwrite;
+
+
+import static net.minecraft.block.FarmlandBlock.setToDirt;
```

Comment:
Code style doesn't permit gratuitous static imports, even in mixins for methods that are "ours".